app/schemas/astrology.py

```
@@ -15,6 +15,44 @@ class APIResponse(BaseModel):
+
+    @validator('birth_time')
+    def validate_birth_time(cls, v):
+        """Valida formato da hora."""
+        try:
+            hour, minute = v.split(':')
+            if len(hour) != 2 or len(minute) != 2:
+                raise ValueError()
+            hour, minute = int(hour), int(minute)
+            if not (0 <= hour <= 23 and 0 <= minute <= 59):
+                raise ValueError()
+            return v
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)